### PR TITLE
Update SIP sign in language

### PR DESCRIPTION
**Pull request by cehsu**
screen shot 2017-09-27 at 11 20 25 am

### CODE REVIEW

**Review by peggygannon — COMMENTED** [review]
LGTM. thanks!

File: src/js/common/schemaform/SaveInProgressIntro.jsx [hunk 1]
```diff
@@ -49,7 +49,7 @@ export default class SaveInProgressIntro extends React.Component {
         <div>
           <div className="usa-alert usa-alert-info schemaform-sip-alert">
             <div className="usa-alert-body">
-              You can save this form, and come back later to finish filling it out. To save your form in progress, please <button className="va-button-link" onClick={() => this.props.toggleLoginModal(true)}>Sign In</button>.
+              You can save this form, and come back later to finish filling it out. <button className="va-button-link" onClick={() => this.props.toggleLoginModal(true)}>Sign in to save your form in progress</button>.
```
Comment (jbalboni) [inline src/js/common/schemaform/SaveInProgressIntro.jsx]: We might want to put a line break in here specifically, so we don't rely on wrapping.

**Review by jbalboni — APPROVED** [review]
LGTM, one minor question

File: src/js/common/schemaform/SaveInProgressIntro.jsx (hunk 1 above)
Comment (cehsu) [reply src/js/common/schemaform/SaveInProgressIntro.jsx]: Great point, added.

**Pull request merged**